### ISSUE TITLE
SimpleDateFormat - parsing time only wrong on iOS

### ISSUE THREAD

**Issue by pedja1** [google/j2objc#771]
For some reason `SImpleDateFormat ` returns different outputs on Android and iOS

Example:

````java
SimpleDateFormat sdf = new SimpleDateFormat("HH:mm:ss");

String time = "08:00:00";

System.out.println(sdf.parse(time));
````

On `Android` this will return `25200000` which is ok.
On `iOS` it returns `21600000` which is wrong.

**Comment by lukhnos** [comment]
This is fixed in current source as of fa63741b02e355ebeae7755ef2efece71acde767.

**Comment by tomball** [comment]
Here are instructions for [how to build J2ObjC from source](http://j2objc.org/docs/Building-J2ObjC.html). We also plan on releasing a new distribution soon.
